openid-provider-commands-1_0.md

```diff
@@ -550,7 +559,7 @@ Following is a non-normative example of Command Response for a Metadata Command:
     "iss": "https://op.example.org",
     "tenant":"73849284748493"
   },
-  "commands_uri": "https://rp.example.net/commands",
+  "command_endpoint": "https://rp.example.net/commands",
```

Comment:
```suggestion
  "command_endpoint": "https://rp.example.net/command",
```

Singular https://rp.example.net/command would be is a bit choice is this example.